src/bootstrap/src/lib.rs

```diff
@@ -641,9 +641,9 @@ impl Build {
         if self.config.profiler_enabled(target) {
             features.insert("profiler");
         }
-
-        // If zkvm target, generate memcpy, etc.
-        if target.contains("zkvm") {
+        // Generate memcpy, etc.  FIXME: Remove this once compiler-builtins
+        // automatically detects this target.
+        if target.contains("zkvm") || target.contains("vex") {
```

Comment:
Suggestion [BOOTSTRAP-CLEANUP 2/3]: can you backlink to the cleanup issue as mentioned in "BOOTSTRAP-CLEANUP 1/3"?